updated public trust principle (#538)
sa-tre/satre-specification: docs/source/principles.md

### Statement

TREs holding public data should build and maintain the trust of data subjects and any other individuals, groups, communities and organisations impacted by the research conducted using TREs. TREs should demonstrate trustworthiness by protecting privacy, keeping data secure and being transparent about their work.

### Rationale

> To ensure continued public support both for the use of data and TREs to facilitate research, it is vital to maintain public trust in the ways TREs hold and use data so this data can continue to be used to deliver public benefits. This could include maintaining the trust of members of the public whose data is held, those who are impacted by research conducted using TREs.

Reviewer: ```suggestion
It is vital to maintain public trust in the ways TREs hold and use data so that this data can be used to deliver public benefits. This could include maintaining the trust of members of the public whose data is held, and those who are impacted by research conducted using TREs.
```
I think this is clearer, the original sentence is a bit circular.

Trust is fragile and can be eroded by data breaches, opaque decision-making, or the perception that data is being used in ways that were not agreed upon or do not serve the public interest.

Public engagement work has indicated there is support for the use of regulated and ethical TREs working for the public benefit as long as conditions are met surrounding security and transparency. Consulting impacted parties, including the public, can help ensure TREs are being used for positive, impactful and agreed-upon purposes. Being transparent about the data held and the projects or organisations who access the data can also help maintain trust.

### Implications

> - Transparency is key to building trust. TREs holding public data should be transparent about the TRE operations, data held and research projects accessing data. This information should be made publicly available in plain language and accessible format.

Reviewer 2: "and an accessible format".  Insert an as it reads better

Reviewer 3: That's fine with me

Reviewer: @gordonmilligan-uod I've added that, do you want to hit merge? 😄

> - Transparency is key to building trust. TREs holding public data should be transparent about the TRE operations, data held and research projects accessing data. This information should be made publicly available in plain language and accessible format.

Reviewer: ```suggestion
- Transparency is key to building trust. TREs holding public data should be transparent about the TRE operations, data held and research projects accessing data. This information should be made publicly available in plain language and an accessible format.
```
